packages/react-frontend/src/SearchPage.jsx

```diff
@@ -3,9 +3,12 @@ import React, { useState, useEffect } from "react";
 function ChefCard({ chef }) {
   return (
     <div className="card">
-      <h2>
-        {chef.firstName} {chef.lastName}
-      </h2>
+      <div className="card-header">
+        <h2>
+          {chef.firstName} {chef.lastName}
+        </h2>
+        <img src={chef.profilePicture} alt="No profile picture" className="profile-picture" />
```

Comment:
just fyi in frontend the className is "chef-image" as it is named in the css file